doc/users/input.rst

```diff
@@ -24,7 +24,7 @@ The ``run_pcm`` executable has to be compiled for a standalone run with:
 
 .. code-block:: bash
 
-   python go_pcm.py -x molecule.inp
+   python <build-path/bin>/go_pcm.py --exe <build-path/bin> --inp molecule.inp 
```

Comment:
It is possible for the script and the executable to be in two different directories

Reply:
Yes, seems so. I just tested it by moving the script from `bin/` to my run folder. Should I just remove the first `<build-path/bin>`?

Reply:
Either that or add a "Note that the script and executable need not be in the same directory"